pkg/client/src/app/pages/applications/manage-imports/manage-imports.tsx

```diff
@@ -148,8 +148,14 @@ export const ManageImports: React.FC = () => {
       cellTransforms: [truncate],
     },
     { title: t("terms.status"), transforms: [cellWidth(10)] },
-    { title: t("terms.accepted"), transforms: [cellWidth(10)] },
```

Comment:
I couldn't set less than 10 to the transforms[cellWidth], the 10 value overlapped with the label, after removal looked the same even after resizing window. Please recommend better solution if this would't be accepted, thanks!